tools/testsys-config/src/lib.rs

```diff
@@ -235,7 +242,9 @@ where
 pub struct TestsysImages {
     pub eks_resource_agent_image: Option<String>,
     pub ecs_resource_agent_image: Option<String>,
+    pub vsphere_k8s_cluster_resource_agent_image: Option<String>,
     pub ec2_resource_agent_image: Option<String>,
+    pub vsphere_vm_resource_agent_image: Option<String>,
```

Comment:
nit: ESXi is more the counterpart to EC2 here:
```suggestion
    pub esxi_resource_agent_image: Option<String>,
```

Reply:
`vsphere_vm_resource_agent_image` is the name used in the TestSys repo and the name of the repo the image is stored in. Do we want to have this deviation?